### ISSUE TITLE
Push a build into the gh-pages branch for a live preview

### ISSUE THREAD

**Issue by Zaid-Ajaj** [fable-elmish/sample-react-counter-list#4]
Use `gh-pages` branche as a place to host a demo:

![untitled](https://cloud.githubusercontent.com/assets/13316248/25091958/ec6cf1d6-238b-11e7-81e5-712703f3f7d1.png)

I will update the other samples shortly to use this as well

**Comment by et1975** [comment]
I'd suggest making `gh-pages` branch and committing the docs output there.

**Comment by Zaid-Ajaj** [comment]
Yes that worked too, I didn't see the option so I assumed is it not supported any more. We will use that
